recipe/meta.yaml

```
@@ -17,26 +17,27 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
-    - poetry >=1.0.5
+    - python >=3.9,<3.13
+    - setuptools >=1.0.5
```

The version is not right, I think.

```suggestion
    - setuptools >=70
```

As per skpro repo, it should be >=61, but based on CVE reports it's better to make it >=70.

https://security.snyk.io/package/pip/setuptools